libs/map/src/lib/components/layer-switcher/stories/vl-map-layer-switcher.stories-templates.ts

```diff
@@ -0,0 +1,71 @@
+import { html } from 'lit-html';
+
+export const kaartLagenOmAanTePassen = () => {
```

Comment:
Waarom hier Nederlands?

Reply:
vermoed dat m'n hersenen in NL-mode waren na iets in storybook gewijzigd te hebben
> aangepast